dataset-spec/README.md

```diff
@@ -0,0 +1,103 @@
+| title       | string            | A short descriptive one-line title for the dataset.          |
+| description | string            | **REQUIRED.** Detailed multi-line description to fully explain the entity. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| keywords    | [string]          | List of keywords describing the dataset.                     |
+| version     | string            | Version of the dataset. [Semantic Versioning (SemVer)](https://semver.org/) SHOULD be followed. |
+| license     | string            | **REQUIRED.** Dataset's license(s) as a [SPDX License identifier or expression](https://spdx.org/licenses/) or `proprietary` if the license is not on the SPDX license list. Proprietary licensed data SHOULD add a link to the license text, see the `license` relation type. |
+| provider    | [Provider Object] | Data provider, the organizations which influenced the content of the dataset. |
+| host        | Host Object       | Storage provider, the organization that hosts the dataset.   |
+| extent      | [Extent Object]   | **REQUIRED.** Spatial and temporal extents.                  |
+| links       | [Link Object]     | **REQUIRED.** A list of references to other documents.       |
+
+### Extent Object
+
+The object describes the spatio-temporal extents of the dataset. Both spatial and temporal extents are required to be specified.
+
+**Note:** STAC datasets tries to be compliant to [WFS 3.0](https://github.com/opengeospatial/WFS_FES), but there are still issues to be solved. The WFS specification is in draft state any may change, especially regarding [3D support](https://github.com/opengeospatial/WFS_FES/issues/143) for spatial extents or the handling of [open date ranges](https://github.com/opengeospatial/WFS_FES/issues/155) for temporal extents. Therefore, It is also likely that the following fields change over time.
+
+| Element  | Type     | Description                                                  |
+| -------- | -------- | ------------------------------------------------------------ |
+| spatial  | [number] | **REQUIRED.** Potential *spatial extent* covered by the dataset. West, north, east, south edges of the spatial extent. Only WGS84 longitude/latitude is supported. The list of four numbers can be extended to six numbers to support a 3D spatial extent. |
+| temporal | string   | **REQUIRED.** Potential *temporal extent* covered by the dataset. Date/time intervals MUST be formatted according to ISO 8601. Open date ranges are supported by omitting either the start or the end time. Example for data from the beginning of 2019 until now: `2009-01-01T00:00:00Z/`. |
+
+### Provider Object
+
+The object provides information about a provider. A provider is any of the organizations that created or processed the content of the dataset and therefore influenced the data offered by this dataset.
```

Comment:
I think you said somewhere that multiple providers are allowed? If so I'd make it a bit more explicit here. If not then we should provide guidance on which one of the 'any...that created or processed'.

Reply:
The Provider object provides information about a single provider, but the Dataset object can hold multiple providers in an array. I made that more clear in the provider field.

Reply:
Makes sent, sounds great.